applications/FluidDynamicsApplication/custom_elements/qs_vms.cpp

```diff
@@ -598,6 +599,53 @@ void QSVMS<TElementData>::AddMassStabilization(
     }
 }
 
+template <class TElementData>
+void QSVMS<TElementData>::AddBoundaryIntegral(TElementData& rData,
+    const Vector& rUnitNormal, MatrixType& rLHS, VectorType& rRHS) {
+
+    constexpr std::size_t StrainSize = (Dim-1)*3;
+    boost::numeric::ublas::bounded_matrix<double,StrainSize,LocalSize> strain_matrix = ZeroMatrix(StrainSize,LocalSize);
```

Comment:
Just wondering, is it necessary to write boost::numeric::ublas::(whatever)? As far as I understand, if we create our own Kratos bounded_matrix we will need to change these lines.